samples/ActionConstraintSample.Web/project.json

```diff
@@ -12,9 +12,12 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "frameworks": {
-    "net451": { },
-    "dnxcore50": {
-      "imports": "portable-net451+win8",
+    "net451": {},
+    "netstandardapp1.5": {
```

Comment:
`netstandardapp1.5` versus `netstandard1.3` seems a bit arbitrary. The samples are used as libraries when invoked from the functional test project. So why isn't `netstandard1.3` also listed?

Similarly all test projects are treated as libraries when invoked through `xunit.console.exe`, `dotnet test` or even `dnx test`.

Guess I'm wondering why the app versus not facet isn't implicit in `"emitEntryPoint": true` or not.